### PR TITLE
Added support for Salus SP600 Smart Plug

### PR DESCRIPTION
**Pull request by MarkAdamson**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -2216,6 +2216,28 @@ const devices = [
...
     },
+
+    // Salus
+    {
+        zigbeeModel: ['SP600'],
+        model: 'SP600',
+        vendor: 'Salus',
+        description: 'Smart plug',
+        supports: 'on/off, power measurement',
+        fromZigbee: [fz.generic_state, fz.ignore_onoff_change, fz.EDP_power, fz.ignore_metering_change],
```
Comment (Koenkk) [inline devices.js]: Could you rename the `EDP_power` converter to `generic_power`?

Comment (MarkAdamson) [reply devices.js]: sure, no problem.

**Comment by MarkAdamson** [comment]
Renamed `EDP_power` converter to `generic_power` :)

**Comment by Koenkk** [comment]
Thanks!

**Comment by davet2001** [comment]
Thanks for submitting this, I am using it.
Before I submit a bug, I seem to be having the SP600 report 1/10th of the expected power, i.e. if something consumes 1234W, it reports it as 123.4W. 

@MarkAdamson does it work ok for you?

**Comment by davet2001** [comment]
Another user has already reported this issue and raised a ticket here:
https://github.com/Koenkk/zigbee2mqtt/issues/2233

**Pull request merged**